client/app/components/Modal/index.js

```diff
@@ -0,0 +1,152 @@
+                    <div className="field">
+                    <p className="control is-expanded has-icons-left has-icons-right">
+                        <input className="input" type="text" placeholder="Company" />
+                        <span className="icon is-small is-left">
+                            <i className="fa fa-building"></i>
+                        </span>
+                    </p>
+                    </div>
+                </div>
+            </div>
+
+            <div className="field is-horizontal">
+                <div className="field-label is-normal">
+                    <label className="label">Contact: </label>
+                </div>
+                <div className="field-body">
+                    <div className="field">
+                    <p className="control is-expanded has-icons-left">
+                        <input className="input" type="tel" placeholder="Your phone number" />
+                        <span className="icon is-small is-left">
+                            <i className="fa fa-phone"></i>
+                        </span>
+                    </p>
+                    </div>
+                    <div className="field">
+                    <p className="control is-expanded has-icons-left has-icons-right">
+                        <input className="input" type="email" placeholder="Email" />
+                        <span className="icon is-small is-left">
+                            <i className="fa fa-envelope"></i>
+                        </span>
+                    </p>
+                    </div>
+                </div>
+            </div>
+                <div className="field is-horizontal">
+                <div className="field-label is-normal">
+                    <label className="label">Subject: </label>
+                </div>
+                <div className="field-body">
+                    <div className="field">
+                    <div className="control">
+                        <input className="input" type="text" placeholder="e.g. Partnership opportunity, Quote Request" />
+                    </div>
+                    </div>
+                </div>
+                </div>
+
+                <div className="field is-horizontal">
+                <div className="field-label is-normal">
+                    <label className="label">Question: </label>
+                </div>
+                <div className="field-body">
+                    <div className="field">
+                    <div className="control">
+                        <textarea className="textarea" placeholder="How can we help you"></textarea>
+                    </div>
+                    </div>
+                </div>
+                </div>
+
+                <div className="field is-horizontal">
+                <div className="field-label">
+                    {/* <!-- Left empty for spacing --> */}
+                </div>
+                <div className="field-body">
```

Comment:
@leseanbiz Check the Bulma docs for better html and styling:

https://bulma.io/documentation/elements/button/#button-group